Error while creating JPAB database

I get the following error when creating the JPAB db on a Linux system with MySQL as the DBMS engine.


[root@vm-arunm oltpbench]# ./oltpbenchmark -b jpab -c ./config/sample_jpab_config.xml --create=true --load=true --execute=true -s 5 -o outputfile
16:30:20,699 (DBWorkload.java:262) INFO  - ======================================================================

Benchmark:     JPAB {com.oltpbenchmark.benchmarks.jpab.JPABBenchmark}
Configuration: ./config/sample_jpab_config.xml
Type:          MYSQL
Driver:        com.mysql.jdbc.Driver
URL:           jdbc:mysql://127.0.0.1:4006/jpab
Isolation:     TRANSACTION_READ_COMMITTED
Scale Factor:  1.0

16:30:20,702 (DBWorkload.java:263) INFO  - ======================================================================
16:30:20,771 (DBWorkload.java:522) INFO  - Creating new JPAB database...
16:30:21,048 (BenchmarkModule.java:200) ERROR - Failed to find DDL file for jpab
Exception in thread "main" java.lang.RuntimeException: Unexpected error when trying to create the jpab database
        at com.oltpbenchmark.api.BenchmarkModule.createDatabase(BenchmarkModule.java:275)
        at com.oltpbenchmark.api.BenchmarkModule.createDatabase(BenchmarkModule.java:255)
        at com.oltpbenchmark.DBWorkload.runCreator(DBWorkload.java:808)
        at com.oltpbenchmark.DBWorkload.main(DBWorkload.java:523)
Caused by: java.lang.NullPointerException
        at com.oltpbenchmark.util.ScriptRunner.runScript(ScriptRunner.java:95)
        at com.oltpbenchmark.api.BenchmarkModule.createDatabase(BenchmarkModule.java:273)
        ... 3 more

--- a/src/com/oltpbenchmark/benchmarks/jpab/ddls/jpab-ddl.sql
+++ b/src/com/oltpbenchmark/benchmarks/jpab/ddls/jpab-ddl.sql
@@ -1,0 +1,3 @@
+---
+--- This File Left Intentionally Blank!
+---

